Packs/CommonTypes/ReleaseNotes/2_9_7.md

```diff
@@ -0,0 +1,36 @@
+#### Indicator Fields
+- **Is Malware Family**
+- **Tool Types**
+- **Sophistication**
+- **Roles**
+- **Tool Version**
+- **Threat Actor Types**
+
+#### Indicator Types
+- **Tool**
+- **Malware**
+- **Report**
+- **Threat Actor**
+- **Attack Pattern**
```

Comment:
#### Indicator Types Changes:
For the following indicator types, custom content that relay on one or more of the types below should be updated to use the new types **after upgrade to Cortex XSOAR version 6.2.0 or higher**.

Reply:
```suggestion

- **Attack Pattern**

#### Indicator Types Changes:

For the following indicator types, custom content that relies on one or more of the following types should be updated to use the new types **after upgrade to Cortex XSOAR version 6.2.0 or higher**.
```